File: connection-examples/deno/insert.ts
import { Client } from "https://deno.land/x/postgres/mod.ts";

const client = new Client({
    user: "MATERIALIZE_USERNAME",
    database: "materialize",
    password: "APP_SPECIFIC_PASSWORD",
    hostname: "MATERIALIZE_HOST",
    port: 6875,
    ssl: true,
})

const main = async ({ response }: { response: any }) => {
    try {
        await client.connect()

        await client.queryObject(
            "INSERT INTO countries(code, name) VALUES($1, $2)",
            ['GH', 'GHANA'],
        );

        const result = await client.queryObject("SELECT * FROM countries")
        console.log(result.rows)
    } catch (err) {
        console.error(err.toString())
    } finally {
        await client.end()
    }
}
export { main }

// Call the main function
main({ response: {} })

File: connection-examples/python/pg8000/insert.py
#!/usr/bin/env python3

import pg8000.native
import ssl

ssl_context = ssl.create_default_context()

conn = pg8000.connect(host="MATERIALIZE_HOST", port=6875, user="MATERIALIZE_USERNAME", password="MATERIALIZE_PASSWORD", database="materialize", ssl_context=ssl_context)

# Insert data
print('Insert data')
insert_query = "INSERT INTO countries (name, code) VALUES ('United States', 'US')"
conn.run(insert_query)
insert_query = "INSERT INTO countries (name, code) VALUES ('Canada', 'CA')"
conn.run(insert_query)
print('Select data')
countries = conn.run("SELECT * FROM countries;")
print(countries)